test/kubernetes/kubernetes.go

```diff
@@ -119,3 +135,121 @@ func CleanupNamespaces(c kubernetes.Interface) error {
 
 	return nil
 }
+
+// NewIngress creates a new ingress
+func NewIngress(c kubernetes.Interface, ingress *networking.Ingress) error {
+	if ingress.Namespace != metav1.NamespaceNone && ingress.Namespace != metav1.NamespaceDefault {
+		return fmt.Errorf("Ingress definitions in the default namespace are not allowed")
```

Comment:
do we really need this guard? creating in the default namespace is not expected for the test, but is still ok in general?